Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -30,6 +30,104 @@ public typealias ContainerViewController = UIViewController & NavigationComponen
  [Pricing Guide](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/).
  */
 open class NavigationViewController: UIViewController, NavigationStatusPresenter, NavigationViewData {
+    
+    // MARK: Accessing NavigationView
+    
+    /**
+     The `NavigationMapView` displayed inside the view controller.
+     
+     - note: Do not change `NavigationMapView.delegate` property; instead, implement the corresponding methods on `NavigationViewControllerDelegate`.
+     */
+    @objc public var navigationMapView: NavigationMapView? {
+        get {
+            return navigationView.navigationMapView
+        }
+    }
+    
+    func setupNavigationCamera() {
+        if let centerCoordinate = navigationService.routeProgress.route.shape?.coordinates.first {
+            navigationMapView?.setInitialCamera(centerCoordinate)
+        }
+        
+        // By default `NavigationCamera` in active guidance navigation should be set to `NavigationCameraState.following` state.
+        navigationMapView?.navigationCamera.follow()
+    }
+    
+    var mapTileStore: TileStoreConfiguration.Location? {
+        NavigationSettings.shared.tileStoreConfiguration.mapLocation
+    }
+        
+    var navigationView: NavigationView {
+        return (view as! NavigationView)
+    }
+    
+    /**
+     Controls whether the main route style layer and its casing disappears
+     as the user location puck travels over it. Defaults to `false`.
+     
+     If `true`, the part of the route that has been traversed will be
+     rendered with full transparency, to give the illusion of a
+     disappearing route. To customize the color that appears on the
+     traversed section of a route, override the `traversedRouteColor` property
+     for the `NavigationMapView.appearance()`.
+     */
+    public var routeLineTracksTraversal: Bool {
+        get {
+            navigationMapView?.routeLineTracksTraversal ?? false
+        }
+        set {
+            navigationMapView?.routeLineTracksTraversal = newValue
+            routeOverlayController?.routeLineTracksTraversal = newValue
+        }
+    }
+    
+    // MARK: Configuring Voice Instructions
```

Comment:
For consistency with some other APIs, though elsewhere we’re a bit inconsistent in this regard:

```suggestion
    // MARK: Configuring Spoken Instructions
```